index.html

```diff
@@ -39,24 +39,30 @@
   <body>
     <header>
       <a class="logo" href="#home">My Logo</a>
-      <nav>
-        <ul class="nav-links">
-          <li>
-            <i id="hamerberg" class="fa-sharp fa-solid fa-bars"></i>
-          </li>
-          <li class="disapear">
-            <a href="#home">Portfolio</a>
-          </li>
-          <li class="disapear">
-            <a href="#aboutme">About</a>
-          </li>
-          <li class="disapear">
-            <a href="#getintouch">Contact</a>
-          </li>
-          <li class="disapear">
-            <a href="#getintouch"><img class="menubtn" src="img/menu.png" alt="email" /></a>
-          </li>
-        </ul>
+      <nav class="navbar">
+          <ul class="nav-menu">
+            <li class="disapear nav-item">
+              <a href="#home" class="nav-link">Portfolio</a>
+            </li>
```

Comment:
- [ ] Just a minor change here: please make sure you display the correct section of your page upon clicking the "Portfolio" link, it is meant to display your projects! 🤓 🔧